packages/client/src/components/common/modals/similar-charges-modal.tsx

```
@@ -0,0 +1,412 @@
+  id: string;
+  chargeType: string;
+  counterpartyName?: string;
+  date?: Date;
+  amountRaw?: number;
+  amountFormatted?: string;
+  vatAmountRaw?: number;
+  vatAmountFormatted?: string;
+  description?: string;
+  tags: string[];
+  taxCategoryName?: string;
+  businessTripName?: String;
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)